openedx/adg/lms/webinars/admin.py

```diff
@@ -247,7 +248,11 @@ def has_change_permission(self, request, obj=None):
         return False
 
     def has_delete_permission(self, request, obj=None):
-        return False
+        """
+        Overriding this method to give delete permission to only those admins
+        belonging to 'Delete Webinar Registrations' group.
```

Comment:
Actually i wanted to say that we should describe something like why do we need to give permissions to delete this. Like we can say, `To delete user objects we need to remove their related registrations and currently admins do not have permissions to delete registrations. To be able to delete user we need to give permission to a specific group of users.`